### PR TITLE
align comments + fix error in stack docs

### PR DESCRIPTION
**Pull request by holiman**

### CODE REVIEW

File: src/main.etk [hunk 1]
```diff
@@ -3,7 +3,7 @@
         push1 0
 %end
 
-%def historical_roots_modulus()
+%def roodmod()
```
Comment (holiman) [inline src/main.etk]:
```suggestion
%def rootmod()
```

File: src/main.etk [hunk 2]
```diff
@@ -37,21 +37,21 @@ push1 revert    # [revert_addr, calldatasize < 32]
 jumpi           # []
 
 # Load stored timestamp.
-push3 historical_roots_modulus() # [hrm, time]
+push3 rootmod() # [hrm, time]
```
Comment (holiman) [inline src/main.etk]:
```suggestion
push3 rootmod() # [hrm]
```

File: src/main.etk [hunk 3]
```diff
@@ -37,21 +37,21 @@ push1 revert    # [revert_addr, calldatasize < 32]
 jumpi           # []
 
 # Load stored timestamp.
-push3 historical_roots_modulus() # [hrm, time]
+push3 rootmod() # [hrm]
 %get_input()    # [time]
```
Comment (holiman) [inline src/main.etk]: This is wrong, should be 
```suggestion
%get_input()    # [time, hrm]
```

File: src/main.etk [hunk 4]
```diff
@@ -37,21 +37,21 @@ push1 revert    # [revert_addr, calldatasize < 32]
...
-%get_input() # [got, want, time_index]
+%get_input()    # [got, want, time_index]
 eq              # [got == want, time_index]
 iszero          # [got != want, time_index]
 push1 revert    # [revert_addr, got != want, time_index]
 jumpi           # [time_index]
 
 # Extend index to get root index.
-push3 historical_roots_modulus() # [hsm, time_index]
+push3 rootmod() # [hsm, time_index]
```
Comment (holiman) [inline src/main.etk]:
```suggestion
push3 rootmod() # [hrm, time_index]
```

File: src/main.etk [hunk 5]
```diff
@@ -75,7 +75,7 @@ sstore                  # []
 %push0()                # [0]
 calldataload            # [root]
 %get_timestamp_index()  # [time_index, root]
-push3 historical_roots_modulus() # [hsm, time_index, root]
+push3 rootmod()         # [hsm, time_index, root]
```
Comment (holiman) [inline src/main.etk]:
```suggestion
push3 rootmod()         # [hrm, time_index, root]
```

**Review by lightclient — APPROVED** [review]

**Pull request merged**